examples/storybook/src/stories/ClearRefinements.stories.ts

```diff
@@ -0,0 +1,30 @@
+import { storiesOf } from '@storybook/angular';
+import { wrapWithHits } from '../wrap-with-hits';
+import meta from '../meta';
+
+storiesOf('ClearRefinements', module)
+  .addDecorator(meta)
+  .add('default', () => ({
+    component: wrapWithHits({
+      template: '<ais-clear-refinements></ais-clear-refinements>',
+      searchParameters: {
+        disjunctiveFacetsRefinements: { brand: ['Apple'] },
+        disjunctiveFacets: ['brand'],
+      },
+    }),
+  }))
+  .add('with nothing to clear', () => ({
+    component: wrapWithHits({
+      template: '<ais-clear-refinements></ais-clear-refinements>',
+    }),
+  }))
+  .add('with clear refinements and query', () => ({
+    component: wrapWithHits({
+      template:
+        "<ais-clear-refinements [clearsQuery]='true'></ais-clear-refinements>",
+      searchParameters: {
```

Comment:
With a refinement widget it would be easier to test (see comment about `wrap-with-hits`).

Reply:
done! d06c4a1